go/vt/vttablet/onlineddl/vrepl.go

```diff
@@ -619,6 +620,7 @@ func (v *VRepl) analyzeBinlogSource(ctx context.Context) {
 		SourceUniqueKeyColumns:       encodeColumns(&v.chosenSourceUniqueKey.Columns),
 		TargetUniqueKeyColumns:       encodeColumns(&v.chosenTargetUniqueKey.Columns),
 		SourceUniqueKeyTargetColumns: encodeColumns(v.chosenSourceUniqueKey.Columns.MappedNamesColumnList(v.sharedColumnsMap)),
+		ForceUniqueKey:               url.QueryEscape(v.chosenSourceUniqueKey.Name),
```

Comment:
I'm curious why we're doing URL escaping?

Reply:
It's for when the index name has spaces, or quotes, or other special characters. The query hints parser will otherwise be unable to parse it correctly. Consider `select /*vt+ ukForce="an index with spaces" */` -- the parser will only parse `"an"`.